trace-callbacks.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2022 All rights reserved.
+# SPDX-License-Identifier: GPL-2.0-only
+# File: trace-callbacks.py
+# Purpose: Trace every N'Siq CppStyle callback for one or more given input source 
+#          files.  Using the trace, the rule developer can plan how to write a new rule.
+import subprocess
+import sys
+import os
+
+if len(sys.argv) == 1:
+    print('Error: Provide one or more source file paths for analysis')
+    print('')
+    print('Example: python trace-callbacks.py "c:\\test\\a.cpp"')
+    print('Example: python trace-callbacks.py "c:\\test\\a.cpp" "d:\\src\\b.cpp"')
+    sys.exit(0)
+
+argument_list = [sys.executable, 'nsiqcppstyle.py', '--filter-string="~ TOOL_trace_nsiqcppstyle_callbacks"', '-o', os.devnull]
+argument_list.extend(sys.argv[1:])
+subprocess.run(argument_list)
```

Comment:
Needs EOF \n

Reply:
Why? The script does not perform a read operation, so there's no fear of waiting on input. In general, please take the time to add a few more words to explain your thinking.

Reply:
Only after two days did the penny drop.  I thought your comment was that run() needed an additional EOF/LF argument.  Now I realize you just wanted an additional LF at the end of the script.  I made the change and pushed.

So my request: if there's any chance of ambiguity, take a few seconds to say something like "add LF at end of script's last line".

OK, I think that's it, awaiting your final clearance and merge.